h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -4456,8 +4456,11 @@ def assertCoefDictEqual(regCoeff, coeff, tol=1e-6):
         assert diff < tol, "diff {0} exceeds tolerance {1}.".format(diff, tol)
 
 
-def assert_equals(expected, actual, message=""):
-    assert expected == actual, ("{0}\nexpected:{1}\nactual\t:{2}".format(message, expected, actual))
+def assert_equals(expected, actual, message="", delta=-1):
```

Comment:
if the default value of delta would be 0, then you cannot use if-else. WDYT?

```suggestion
def assert_equals(expected, actual, message="", delta=0):
```